crates/sozo/ops/src/auth.rs

```diff
@@ -115,7 +121,7 @@ where
         let resource_selector =
             get_resource_selector(ui, world, &new_writer.resource, default_namespace).await?;
         let contract_address =
-            utils::get_contract_address(world, new_writer.tag_or_address).await?;
+            utils::get_contract_address(world, new_writer.tag_or_address.clone()).await?;
```

Comment:
`get_contract_address` should take a `&str` instead, can be done later though.

Reply:
will do in this PR